### ISSUE TITLE
latexpdf backend does not support document subtitle

### ISSUE THREAD

**Issue by getreu** [sphinx-doc/sphinx#3567]
### latexpdf backend does not support document subtitle 

The issue and some workaround was first described in this post: 
[How do I add a subtitle](https://groups.google.com/forum/#!topic/sphinx-users/yVcIsn_Hc24)

Consider the following ReStructuredText input

```
*****
Title
*****

----------------------
Subtitle
----------------------

.. sectnum::

Section
=======

Subsection
----------

body
```

It renders to 

![screenshot_2017-03-18_17-24-02](https://cloud.githubusercontent.com/assets/579082/24073409/fc83f004-0bff-11e7-80a3-0cbb0cc827b7.png)

Problem: the latexpdf writer in sphinx does not support the ``subtitle`` (the html writer does).

**Comment by tk0miya** [comment]
What did you mean "does not support the subtitle"? Please let me in detail.
Certainly, Sphinx skips first section title (in this case, "Title"). But following subtitle is rendered as PDF. So I can't understand your problem.

**Comment by getreu** [comment]
When I run the following code with `rst2pdf` 
```
*****
Title
*****
----------------------
Subtitle
----------------------

.. sectnum::

Section
=======

Subsection
----------

body
```
I get the desired result:
![screenshot_2017-03-19_12-18-46](https://cloud.githubusercontent.com/assets/579082/24080088/433a0150-0ca0-11e7-8e09-6a09edff1ae5.png)
The string `Subtitle` is placed under the string `Title` on the title page with a smaller font.

Now I try with Sphinx:

This is my input 

index.rst
```
********
Title
********

------------
Subtitle
------------


.. toctree::
   :maxdepth: 2
   :caption: Contents:

   body
```

body.rst
```
Section
=============

Subsection
---------------

body
```

`make html` is almost ok (see @jfbu 's comment below):
![459ede0c-0ca0-11e7-89f6-c4d5d53fe2a5](https://cloud.githubusercontent.com/assets/579082/24131948/5ee30f72-0dfa-11e7-890b-589f8f313379.png)

but  `make latexpdf` 

gives the following:

![48d77098-0ca0-11e7-8015-a7917506cbce](https://cloud.githubusercontent.com/assets/579082/24132019/e06ff0e6-0dfa-11e7-9082-4f48cb2a14f4.png)

The string `Subtitlte` is mistakenly understood as a section.

**Comment by jfbu** [comment]
The html output is not fully satisfying, why is `subtitle` linked in the sidebar holding the TOC ? it does suggest `subtitle` is a sectioning command, not part of the title.

The PDF title does not come from what is put in the first title, it comes from the `latex_documents` variable, for example:

```
latex_documents = [
    (master_doc, 'FOO.tex', 'FOO Documentation',
     'BAR', 'manual'),
]
```

In theory one should be able to use it this way:
```
latex_documents = [
    (master_doc, 'FOO.tex', 'FOO Documentation\\\\\\huge Sub-title',
     'BAR', 'manual'),
]
```
to generate a LaTeX sub-title. But the ``\huge`` creates an infinite loop when `\@title` macro is expanded in piece of code which attempts to set the PDF info dictionary and is anyhow without effect (see issue #3551). Ironically although having no final effect nevertheless the code in `sphinxmanual.cls` creates the infinite loop.

Waiting for comments on multiple builders sub-title possibility for Sphinx.

At any rate, when #3551 will be fixed, the present issue will be taken into account so that `latex_documents` can be used like the above. 

At a later stage, templating for PDF title page would be nice.

**Comment by getreu** [comment]
@jfbu 

> The html output is not fully satisfying, why is subtitle linked in the sidebar holding the TOC ? it does suggest subtitle is a sectioning command, not part of the title.

I opened #3574 for this

**Comment by tk0miya** [comment]
Thanks for clarification. I understood what you mean.

Unfortunately, Sphinx does not support a "subtitle". Sphinx recognizes it as a mere second level section. So it is appeared in TOC as a section.
In detail, Sphinx always uses `doctitle_xform: False` on parsing reST.
http://docutils.sourceforge.net/docs/user/config.html#doctitle-xform
https://github.com/sphinx-doc/sphinx/blob/1.5.3/sphinx/environment/__init__.py#L59

Thanks,

**Comment by jfbu** [comment]
At 1.5.4 one will be able to do this:

```
latex_documents = [
    (master_doc, 'project.tex', 'Project Documentation\\\\\\LARGE Subtitle',
     'Author', 'manual'),
]
```

(or  ``...\\\\{\\LARGE Subtitle}`` for more spacing between main Title and Subtitle.)

to set-up a subtitle especially for PDF output. As this will be inserted as latex code, recall to use suitable LaTeX escapes (for example for the underscore) and either use raw Python strings or double the backslashes.

This should have been possible earlier (and is now fixed at  895014c) but wasn't due to commands like ``\LARGE`` causing infinite loop in LaTeX in the code doing a (failed) attempt to set up the PDF Document Information Dictionary (issue related to this PDF metadata is discussed at #3551).

Closing this as the general subtitle issue is discussed at #3574. Thanks for reporting.
